src/main/java/frc/robot/RobotContainer.java

```diff
@@ -51,13 +51,6 @@ public RobotContainer() {
               -driveController.getRawAxis(1), -driveController.getRawAxis(5)),
               drive)
     );*/
-
-    drive.setDefaultCommand(
-      new RunCommand(
-        () -> drive.testAnglePIDLoop(drive.frontRightModule,
-         (driveController.getRawAxis(0) * .5), (-driveController.getRawAxis(1)) * .5),
-         drive)
-    );
     
```

Comment:
go ahead and add in a default command here when you move intake from Robot to RobotContainer
`intake.setDefaultCommand(new IntakeCommand(), intake)`
I believe should work